docs/bsp.md

```diff
@@ -1240,13 +1237,23 @@ trait SbtBuildTarget {
   /** The classpath for the sbt build (including sbt jars). */
   def classpath: List[Uri]
   
-  /** The Scala build target associated with this sbt build.
-    * It contains the scala version and the scala jars used. */
+  /** The Scala build target describing the scala
+   * version and scala jars used by this sbt version. */
   def scalaBuildTarget: ScalaBuildTarget
+  
+  /** An optional parent if the target has an sbt meta project. */
+  def parent: Option[BuildTargetIdentifier]
+  
+  /** The build targets defined by the sbt build represented
+    * in this target. It can contain scala targets or sbt build
+    * targets if this target represents an sbt meta-meta build. */
+  def definedTargets: List[Json]
```

Comment:
Can we type this better? For example separate into two fields : definedSbtTargets and definedScalaTargets

Reply:
This should have been `BuildTargetIdentifier` from the start. Now it is, it's more typed and represents better that it can create scala and sbt build targets. I prefer to keep this rather than the two independent fields.